### ISSUE TITLE
support 1.20.4

### ISSUE THREAD

**Issue by seema84** [LordRazen/leather-colorizer#46]
```
[22:35:11] [Server thread/INFO]: Player issued server command: /leathercolorizer:/lc
[22:35:11] [Server thread/ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'leathercolorizer:/lc' in plugin LeatherColorizer v1.1
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[paper-api-1.20.4-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:155) ~[paper-api-1.20.4-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_20_R3.CraftServer.dispatchCommand(CraftServer.java:999) ~[paper-1.20.4.jar:git-Paper-430]
	at org.bukkit.craftbukkit.v1_20_R3.command.BukkitCommandWrapper.run(BukkitCommandWrapper.java:64) ~[paper-1.20.4.jar:git-Paper-430]
	at com.mojang.brigadier.context.ContextChain.runExecutable(ContextChain.java:73) ~[brigadier-1.2.9.jar:?]
	at net.minecraft.commands.execution.tasks.ExecuteCommand.execute(ExecuteCommand.java:32) ~[paper-1.20.4.jar:git-Paper-430]
	at net.minecraft.commands.execution.tasks.ExecuteCommand.execute(ExecuteCommand.java:19) ~[paper-1.20.4.jar:git-Paper-430]
	at net.minecraft.commands.execution.UnboundEntryAction.lambda$bind$0(UnboundEntryAction.java:8) ~[paper-1.20.4.jar:git-Paper-430]
	at net.minecraft.commands.execution.CommandQueueEntry.a(CommandQueueEntry.java:5) ~[paper-1.20.4.jar:git-Paper-430]
	at net.minecraft.commands.execution.ExecutionContext.runCommandQueue(ExecutionContext.java:101) ~[paper-1.20.4.jar:git-Paper-430]
	at net.minecraft.commands.Commands.executeCommandInContext(Commands.java:434) ~[?:?]
	at net.minecraft.commands.Commands.performCommand(Commands.java:336) ~[?:?]
	at net.minecraft.commands.Commands.performCommand(Commands.java:323) ~[?:?]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.performChatCommand(ServerGamePacketListenerImpl.java:2230) ~[?:?]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.lambda$handleChatCommand$14(ServerGamePacketListenerImpl.java:2190) ~[?:?]
	at net.minecraft.util.thread.BlockableEventLoop.lambda$submitAsync$0(BlockableEventLoop.java:59) ~[?:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
	at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper-1.20.4.jar:git-Paper-430]
	at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:153) ~[?:?]
	at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[?:?]
	at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1455) ~[paper-1.20.4.jar:git-Paper-430]
	at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:194) ~[paper-1.20.4.jar:git-Paper-430]
	at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:126) ~[?:?]
	at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1432) ~[paper-1.20.4.jar:git-Paper-430]
	at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1355) ~[paper-1.20.4.jar:git-Paper-430]
	at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:136) ~[?:?]
	at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1333) ~[paper-1.20.4.jar:git-Paper-430]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1222) ~[paper-1.20.4.jar:git-Paper-430]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:319) ~[paper-1.20.4.jar:git-Paper-430]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.lang.ExceptionInInitializerError
	at com.minecraftheads.leathercolorizer.data.InventoryMapping.<clinit>(InventoryMapping.java:15) ~[Leather-Colorizer-1.1.jar:?]
	at com.minecraftheads.leathercolorizer.utils.InventoryCreatorBridge.<init>(InventoryCreatorBridge.java:18) ~[Leather-Colorizer-1.1.jar:?]
	at com.minecraftheads.leathercolorizer.commands.CommandLeatherColorizer.onCommandPlayer(CommandLeatherColorizer.java:66) ~[Leather-Colorizer-1.1.jar:?]
	at com.minecraftheads.leathercolorizer.commands.CommandLeatherColorizer.onCommand(CommandLeatherColorizer.java:37) ~[Leather-Colorizer-1.1.jar:?]
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[paper-api-1.20.4-R0.1-SNAPSHOT.jar:?]
	... 29 more

```

the plugin cannot be compiled either
```
for project com.minecraftheads:Leather-Colorizer:jar:1.2: com.minecr
aftheads:plugin-utils:jar:1.7-SNAPSHOT was not found in https://hub.spigotmc.org
/nexus/content/repositories/snapshots/ during a previous attempt.
```

**Comment by bmack94** [comment]
I updated the docs. To compile it you can follow the docs now: https://github.com/LordRazen/leather-colorizer?tab=readme-ov-file#build-information 
You need to install the mvn dependency: https://github.com/LordRazen/plugin-utils

@LordRazen can you upload the new version in the spigot portal?

**Comment by seema84** [comment]
thanks with the plugin-utils it worked. 
besides spigot it would also be good if you could download the jar here on github (at the release assets)

**Comment by bmack94** [comment]
Good point, i added it to the release

**Comment by bmack94** [comment]
Fixed in Release v1.2.0 https://github.com/LordRazen/leather-colorizer/releases/tag/v1.2.0
